prepline_sec_filings/api/section.py

```diff
@@ -100,17 +100,14 @@ def pipeline_api(text, m_section=[], m_section_regex=[]):
         else:
             m_section = [enum.name for enum in SECTIONS_S1]
     for section in m_section:
-        results[section] = sec_document.get_section_narrative(
-            section_string_to_enum[section]
-        )
+        results[section] = sec_document.get_section_narrative(section_string_to_enum[section])
     for i, section_regex in enumerate(m_section_regex):
         regex_enum = get_regex_enum(section_regex)
         with timeout(seconds=5):
             section_elements = sec_document.get_section_narrative(regex_enum)
             results[f"REGEX_{i}"] = section_elements
     return {
-        section: convert_to_isd(section_narrative)
-        for section, section_narrative in results.items()
+        section: convert_to_isd(section_narrative) for section, section_narrative in results.items()
```

Comment:
It looks like this somehow got formatted by `black` with line length 88 and it's comparing with the version that used line length 100. In any case I don't think this file should change in this PR since the associated notebook hasn't changed, so you can fix the CI problem by reverting the changes to `prepline_sec_filings/api/section.py`.

Reply:
Done